spec-tests/src/test/java/org/candlepin/spec/consumers/ConsumerResourceSpecTest.java

```diff
@@ -1020,6 +1020,28 @@ public void shouldOnlyFindConsumersForGivenEnvironmentIdAndOwner() {
             .doesNotContain(consumer1.getId(), consumer2.getId());
     }
 
+    @Test
+    public void shouldProduceCheckInForCloudConsumer() {
+        ApiClient adminClient = ApiClients.admin();
+
+        OwnerDTO owner = adminClient.owners().createOwner(Owners.random());
+        ConsumerDTO consumer = adminClient.consumers().createConsumer(Consumers.randomAWS(owner));
+
+        OffsetDateTime before = consumer.getLastCheckin();
+
+        // Produce a check-in event
+        ApiClients.trustedConsumer(consumer.getUuid())
```

Comment:
We should avoid using trustedConsumer auth in spec tests (unless we're directly trying to test that authentication method or something relating to it). Lets use ApiClients.ssl() instead

Reply:
Good catch! Updated